lib/pages/input_participant_info/input_participant_info_page.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+import 'package:morning_weakers/pages/input_participant_info/widgets/desired_occupation.dart';
+
+class InputParticipantInfoPage extends StatelessWidget {
+  static const String pageName = '参加者情報入力';
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(pageName),
```

Comment:
ごめん！`static const String pageName = '参加者情報入力';` 
ここ俺が放置してしまっとったけど他のファイルで使ってないけんそのまま入れて大丈夫かも🙏
```suggestion
        title: Text('参加者情報入力'),
```

Reply:
おっけー！確認ありがと！